### ISSUE TITLE
Broken in Firefox

### ISSUE THREAD

**Issue by wilt00** [austin-ek/rosette#27]
In Firefox (v.62), none of the blocks appear, and neither does the central column labelled 'Surus'. 

![image](https://user-images.githubusercontent.com/7883508/44701689-97c4cf80-aa55-11e8-82cc-b07c5d4e1421.png)

The console gives this error:

```
uncaught exception: NumberFormatException: Invalid number format: ''
```

Unrelatedly, I think, it also gives this warning:
```
Source map error: request failed with status 404
Resource URL: https://outreach.chicagoedt.org/robocode/game/js/lib/kotlin.js
Source Map URL: kotlin.js.map
```
Source maps are nice for debugging, but they shouldn't be causing something like this.

It's late tonight, so I might dig into this a little more later. (Most schools don't use Firefox after all)

Just as a side note, I opened the page in Chrome and this looks amazing! Awesome work, you guys.

**Comment by wilt00** [comment]
Just for the heck of it, I also tried it in Edge. Same deal, slightly more helpful error message:

```
SCRIPT5022: SCRIPT5022: NumberFormatException: Invalid number format: ''
kotlin.js (25843,7)
```

**Comment by austin-ek** [comment]
This has been fixed in commit 22e5ee8

**Comment by austin-ek** [comment]
I suspect the source map message is because the kotlin files are not available on the server. I haven't looked at the backend to verify.

We've been discussing changes for the automatic build configuration. I will look into fixing that message then.
